src/Microsoft.DotNet.Interactive.Tests/CompositeKernelTests.cs

```diff
@@ -425,6 +426,189 @@ public async Task Deferred_commands_on_composite_kernel_are_execute_on_first_sub
                     typeof(CommandSucceeded));
         }
 
+        [Fact]
+        public async Task quit_command_cancels_all_deferred_commands_on_composite_kernel()
```

Comment:
These aren't really just tests of the `CompositeKernel`. It might be useful to split them into a new test class and move toward a more vertical test organization.

Reply:
moved to QuiCommandTests class